.github/workflows/uberjar.yml

```diff
@@ -100,6 +100,12 @@ jobs:
         path: bin/docker/
     - name: Move the ${{ matrix.edition }} uberjar to the context dir
       run: mv bin/docker/target/uberjar/metabase.jar bin/docker/.
+    # We need it for multi-arch build
+    - name: Set up QEMU
+      if: ${{ github.ref_name == 'master' }}
+      uses: docker/setup-qemu-action@v2
+      with:
```

Comment:
I don't see this in their documentation, where did you get this flag from?

Reply:
It is a bit not obvious, but it is required based on the Docker documentation for buildX - https://docs.docker.com/build/ci/github-actions/multi-platform/

Reply:
If it works, doesn't hurt my feelings